web/client/plugins/LongitudinalProfileTool.jsx

```diff
@@ -70,6 +70,7 @@ import { createPlugin } from '../utils/PluginsUtils';
  * @prop {Object} cfg.config the plugin configuration
  * @prop {string} cfg.config.wpsurl optional, the geoserver url the the wps endpoint to use. It can be an absolute url. default is "/geoserver/wps".
  * @prop {string} cfg.config.chartTitle the default title of the chart
+ * @prop {number} cfg.config.noDataThreshold the number that beyond it will exclude data
```

Comment:
It is ok to make it not configurable by UI? It depends on the layer, maybe it should be configured for layer or manually inserted.

Reply:
it was requested to develop the quickest solution

Reply:
Ok. If @tdipisa is aware of this. it is ok for me.